configuration/src/main/java/org/ehrbase/configuration/config/security/SecurityConfiguration.java

```diff
@@ -52,6 +53,12 @@ public class SecurityConfiguration {
 
     private final SecurityConfig securityConfig;
 
+    /**
+     * Extended property on spring actuator config that defines who can access the management endpoint.
```

Comment:
incorrect javadoc

Reply:
thx ... was a left over from the old implementation.